overrides.css

```diff
@@ -1,149 +1,9 @@
 /* Deno Doc Overrides */
 
-div.ddoc {
-  @apply flex h-screen p-4 pt-2 lg:p-2 lg:pt-0;
-}
-
-/* First column: fixed width of 290px */
-.ddoc > :first-child {
-  @apply w-[290px] flex-shrink-0;
-}
-
-/* Second column: takes up the remaining space */
-.ddoc > :nth-child(2) {
-  @apply flex-grow;
-}
-
-.toc {
-  @apply hidden;
-}
-
-.ddoc > .toc {
-  @apply hidden md:flex md:flex-col !important;
-}
```

Comment:
The styles here that adjust the ToC position are still necessary because if all of ddoc is in a grid it doesn't anchor the navigation content to the left with a fixed width so I think ddoc should be a flex where the categories list is on the left and the rest of the content takes up the rest of the space. 

We could either keep that in here until the next deno_doc pass or migrate it to styles.css as something that is a deno-docs specific interpretation

<img width="1181" alt="Screenshot 2024-07-11 at 9 05 39 AM" src="https://github.com/denoland/deno-docs/assets/776987/a6a7c72b-043c-46e6-842b-ec0d4e4140ce">